SpatialGDK/Documentation/content/game-client-interest-management.md

```diff
@@ -0,0 +1,115 @@
+<%(TOC)%>
+# Game client interest management
+
+The GDK uses [interest]({{urlRoot}}/content/glossary#interest) to control the information that [game clients]({{urlRoot}}/content/glossary#client-workers) receive about the world around them.
+
+Game clients need information about Actors that they don’t own (see the Unreal documentation on [owning](https://docs.unrealengine.com/en-us/Gameplay/Networking/Actors/OwningConnections) to help them to correctly manipulate the Actors that they _do_ own, and to render the SpatialOS world. Interest enables game clients to receive the relevant information.
+
+You can define your game client’s interest in three ways, which you can use alongside each other:
+
+* **`NetCullDistanceSquared`**<br>
+A property that you define on an Actor’s class. For any Actor of that class or a derived class, game clients whose players are within the distance you specify receive updates about the Actor.<br>
```

Comment:
"A property defined on an Actor's class. For any instance of an Actor of that class or a derived class, game clients whose players are within the distance specified receive updates about the Actor. <br>"

I changed this because this property exists regardless if the developer defines it. Maybe we want to still say something like "you change this" but I think it's important to say that this is just a value tweaked, rather than a value they define and tweak.

Reply:
Nice tech-writer-like attention to the subtleties of the wording - I love it! I'll also update the bit under the NetCullDistancSquared heading, to make it clear that this is already a thing and you can tweak it, rathe than it being a thing you set up.